gpu/gsystem.go

```diff
@@ -228,11 +228,11 @@ func (sy *GraphicsSystem) beginRenderPass() (*Render, *wgpu.TextureView, error)
 // to start the render pass using the Renderer configured for
 // this system, and returns the encoder object to which further
 // rendering commands should be added.
-// Call EndRenderPass when done.
+// Call [EndRenderPass] when done.
 // This version Clears the target texture first, using ClearValues.
 func (sy *GraphicsSystem) BeginRenderPass() (*wgpu.RenderPassEncoder, error) {
 	rd, view, err := sy.beginRenderPass()
-	if err != nil {
+	if errors.Log(err) != nil {
```

Comment:
Is it really good practice to `errors.Log` everywhere, given that these functions return errors that will presumably themselves be `errors.Log`ged?